ui/apps/dashboard/src/layout/index.tsx

```diff
@@ -11,17 +11,39 @@ const { Sider: AntdSider, Content: AntdContent } = AntdLayout;
 
 export const MainLayout: FC = () => {
   const { authenticated } = useAuth();
+  const [collapsed, setCollapsed] = useState(false);
```

Comment:
here, it's implemented the similar logic twice, one in MainLayout, the other one in Sider, I think we can reduce one, only implement logic of checking collapse in the MainLayout, and pass the state of collapse the Sider component:
```
<Sider collapse={size?.width <= 768} />
```